budget_manager/models.py

```diff
@@ -1,3 +1,36 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+
+class Sources(models.Model):
+    source_name = models.CharField(max_length=50)
+    created_by = models.CharField(max_length=50, null=True)
+
+
+class Currencies(models.Model):
+    name = models.CharField(max_length=20)
+    currency_code = models.CharField(max_length=10)
+
+
+class Categories(models.Model):
+    category_name = models.CharField(max_length=50)
+    created_by = models.CharField(max_length=50, null=True)
```

Comment:
to ma być `CharField` na pewno? Ten komentarz tyczy się tez innych miejsc, gdzie tak zostało to zrobione